### PR TITLE
comments: modify comments of src_ino and i_ino

### PR DESCRIPTION
**Pull request by power-more**
Signed-off-by: zhaoshang <zhaoshangsjtu@linux.alibaba.com>

### CODE REVIEW

File: rafs/src/metadata/layout/v5.rs [hunk 1]
```diff
@@ -914,7 +914,7 @@ pub struct RafsV5Inode {
     pub i_digest: RafsDigest, // 32
     /// parent inode number
     pub i_parent: u64,
-    /// from fs stat()
+    /// rafs inode number, set_ino() from RAFS_ROOT_INODE = 1
```
Comment (jiangliu) [inline rafs/src/metadata/layout/v5.rs:917]: The comment is a little confusing, could you help to elaberate more?

**Comment by yqleng1987** [comment]
@power-more , your pull request has been updated. A new test job will be submitted. Please wait in patience.

**Comment by yqleng1987** [comment]
@power-more , the test job has been submitted. Please wait in patience.

File: rafs/src/metadata/layout/v5.rs (hunk 1 above)
Comment (power-more) [reply rafs/src/metadata/layout/v5.rs]: src_ino is rafs inode number, but not from fs stat(). 
src_ino is set by function set_ino(), and its number is from const RAFS_ROOT_INODE = 1.

File: rafs/src/metadata/layout/v5.rs [hunk 2]
```diff
@@ -914,7 +914,7 @@ pub struct RafsV5Inode {
     pub i_digest: RafsDigest, // 32
     /// parent inode number
     pub i_parent: u64,
-    /// from fs stat()
+    /// rafs inode number, set from 1
```
Comment (bergwolf) [inline rafs/src/metadata/layout/v5.rs:917]: How about:
```
/// Artifact inode number set by the nydus image builder. Start from RAFS_ROOT_INODE  = 1.
```

**Comment by yqleng1987** [comment]
@power-more , your pull request has been updated. A new test job will be submitted. Please wait in patience.

File: rafs/src/metadata/layout/v5.rs (hunk 2 above)
Comment (power-more) [reply rafs/src/metadata/layout/v5.rs]: fix

**Review by jiangliu — APPROVED** [review]

**Pull request merged**